docs/boards/Board - CrazyBeeF3FR.md

```diff
@@ -0,0 +1,74 @@
+# CrazyBee F3 FR
+![CrazyBee F3 FR front](https://raw.githubusercontent.com/githubDLG/pic/master/CrazyBeeF3FRtop.jpg)
```

Comment:
It's better to add the images to git and include them in the pull request - linking to images in the wiki means that the contents of the `docs/` folder will be broken if the wiki contents are changed.

Reply:
er...  so , mikeller , what the exactly mean of         add the images to git ..         Since I have already add the images in my git repository :upside_down_face:

Reply:
You must add the image as a file of this PR, and reference it as a relative url in the same repository.

Reply:
@mikeller I don't see similar images, there are images in several folders, docs/Wiring, docs/assets/images, docs/screenshots... what is the best folder? Or he must use a new one?